keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone.py

```diff
@@ -0,0 +1,434 @@
+            consistency with MobileNetV1 in Keras.
+            - If `alpha` < 1.0, proportionally decreases the number
+                of filters in each layer.
+            - If `alpha` > 1.0, proportionally increases the number
+                of filters in each layer.
+            - If `alpha` = 1, default number of filters from the paper
+                are used at each layer.
+        dropout_rate: a float between 0 and 1 denoting the fraction of input
+            units to drop during training, defaults to 0.2.
+
+    Examples:
+    ```python
+    input_data = tf.ones(shape=(8, 224, 224, 3))
+
+    # Randomly initialized backbone with a custom config
+    model = MobileNetV3Backbone(
+        stackwise_expansion=[1, 72.0 / 16, 88.0 / 24, 4, 6, 6, 3, 3, 6, 6, 6],
+        stackwise_filters=[16, 24, 24, 40, 40, 40, 48, 48, 96, 96, 96],
+        stackwise_stride=[2, 2, 1, 2, 1, 1, 1, 1, 2, 1, 1],
+        filters=1024,
+        include_rescaling=False,
+    )
+    output = model(input_data)
+    ```
+    """  # noqa: E501
+
+    def __init__(
+        self,
+        *,
+        stackwise_expansion,
+        stackwise_filters,
+        stackwise_kernel_size,
+        stackwise_stride,
+        stackwise_se_ratio,
+        stackwise_activation,
+        filters,
+        include_rescaling,
+        input_shape=(None, None, 3),
+        input_tensor=None,
+        alpha=1.0,
+        dropout_rate=0.2,
+        **kwargs,
+    ):
+        inputs = utils.parse_model_inputs(input_shape, input_tensor)
+        x = inputs
+
+        if include_rescaling:
+            x = layers.Rescaling(scale=1 / 255)(x)
+
+        x = layers.Conv2D(
+            16,
+            kernel_size=3,
+            strides=(2, 2),
+            padding="same",
+            use_bias=False,
+            name="Conv",
+        )(x)
+        x = layers.BatchNormalization(
+            axis=CHANNEL_AXIS,
+            epsilon=BN_EPSILON,
+            momentum=BN_MOMENTUM,
+            name="Conv/BatchNorm",
+        )(x)
+        x = apply_hard_swish(x)
+
+        pyramid_level_inputs = {}
+        for stack_index in range(len(stackwise_filters)):
+            x = apply_inverted_res_block(
+                x,
+                expansion=stackwise_expansion[stack_index],
+                filters=adjust_channels(
+                    (stackwise_filters[stack_index]) * alpha
+                ),
+                kernel_size=stackwise_kernel_size[stack_index],
+                stride=stackwise_stride[stack_index],
+                se_ratio=stackwise_se_ratio[stack_index],
+                activation=stackwise_activation[stack_index],
+                expansion_index=stack_index,
+            )
+            pyramid_level_inputs[stack_index + 2] = x.node.layer.name
+
+        last_conv_ch = adjust_channels(backend.int_shape(x)[CHANNEL_AXIS] * 6)
+
+        # if the width multiplier is greater than 1 we
+        # increase the number of output channels
+        if alpha > 1.0:
```

Comment:
This block can be removed since we don't have a top.

Reply:
Done